src/main/scala/uk/gov/nationalarchives/fileformat/Lambda.scala

```diff
@@ -12,11 +12,10 @@ import io.circe.parser.decode
 import software.amazon.awssdk.services.sqs.model.{DeleteMessageResponse, SendMessageResponse}
 import uk.gov.nationalarchives.aws.utils.Clients.{kms, sqs}
 import uk.gov.nationalarchives.aws.utils.{KMSUtils, SQSUtils}
-import uk.gov.nationalarchives.fileformat.FFIDExtractor.FFIDFile
+import uk.gov.nationalarchives.fileformat.FFIDExtractor.{FFIDFile, _}
 
```

Comment:
It looks like the extra `_ ` import is unused.

Reply:
Fixed, thanks.